edalize/symbiflow.py

```diff
@@ -148,6 +148,13 @@ def configure_nextpnr(self):
                                  'vivado.sh',
                                  dict())
 
+        vendor = self.tool_options.get('vendor', None)
+
+        source = self.tool_options.get('source', None)
```

Comment:
Need a comment on what source should be https://github.com/SymbiFlow/edalize/blob/5d04788b65d16389f2067ca6da0f4926a2eb60dc/edalize/symbiflow.py#L30-L61